polynote-frontend/polynote/ui/monaco/themes.ts

```diff
@@ -0,0 +1,30 @@
+import {editor} from "monaco-editor";
+import IStandaloneThemeData = editor.IStandaloneThemeData;
+
+export const themes: Record<string, IStandaloneThemeData> = {
+  "light": {
+    base: 'vs',
+    inherit: true,
+    rules: [
+      { token: 'keyword', foreground: '000088', fontStyle: 'bold' },
+      { token: 'string', foreground: '008800', fontStyle: 'bold'},
+      { token: 'number', foreground: '0000FF' }
+    ],
+    colors: {
+      'editor.lineHighlightBackground': '#FFFAE3',
+      'editor.lineHighlightBorder': '#00000000',
+      'editorOverviewRuler.border': "#00000000"
+    }
+  },
+  "dark": {
+    base: 'vs-dark',
+    inherit: true,
+    rules: [
+      { token: 'keyword', fontStyle: 'bold' }
+    ],
+    colors: {}
+  }
```

Comment:
Surprising that `dark` is a lot more sparse than `light` - is that just because we don't like the light colors as much?

Reply:
It's mainly my own bias – I originally wanted the colors to match IntelliJ (though they don't really anymore) but since I don't use dark mode I'm ambivalent about the dark mode colors.